public/index.html

```diff
@@ -28,6 +28,8 @@ <h1>Codice d'Boca</h1>
         <p>Menu Loading&hellip;</p>
       </div>
     </div>
+    <span>this is my change - jonathans</span>
```

Comment:
Hmm... let's leave this off. Also, don't commit .DS_Store files. Add to .gitignore